src/Native/LdaNative/data_block.h

```diff
@@ -44,7 +44,7 @@ namespace lda
         int64_t used_size_;
```

Comment:
>int64_t [](start = 8, length = 7)

should used_size_ also be unsigned?

Reply:
Perhaps, but that would require a lot of changes.  I have only touched those variables that go directly into the size of the array that can under-allocate. In this case, the used_size_ is just a counter variable that measures some document size.  

---
In reply to: [276827553](https://github.com/dotnet/machinelearning/pull/3370#discussion_r276827553) [](ancestors = 276827553)